xwiki-commons-tools/xwiki-commons-tool-xar/xwiki-commons-tool-xar-plugin/src/main/java/org/xwiki/tool/xar/Transformation.java

```diff
@@ -50,7 +50,18 @@
         /**
          * Add passed XML as child of the found node.
          */
-        INSERT_CHILD
+        INSERT_CHILD,
+
+        /**
+         * Add content of the text file as child-text of the found node.
```

Comment:
You might want to add some `@since` to those two new enums.

Reply:
Done.